frontend/src/components/form/BirthField.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <Field class="birth-field">
+    <Label for="year" :required="required">생년월일</Label>
+    <div class="birth">
+      <TextInput
+        v-model="year"
+        class="year"
+        id="year"
+        name="year"
+        type="text"
+        placeholder="YYYY"
+        :required="required"
+      />
+      <TextInput
+        v-model="month"
+        class="month"
+        name="month"
+        type="text"
+        placeholder="MM"
+        :required="required"
+      />
+      <TextInput
+        v-model="day"
+        class="day"
+        name="day"
+        type="text"
+        placeholder="DD"
+        :required="required"
+      />
+    </div>
+    <RuleField v-show="incorrectYear" v-model="incorrectYear" :rules="yearRules" :target="year" />
+    <RuleField
+      v-show="!incorrectYear && incorrectMonth"
+      v-model="incorrectMonth"
+      :rules="monthRules"
+      :target="month"
+    />
+    <RuleField
+      v-show="!incorrectYear && !incorrectMonth && incorrectDay"
+      v-model="incorrectDay"
+      :rules="dayRules"
+      :target="day"
+    />
+  </Field>
+</template>
+
+<script>
+import Label from "./Label"
+import Field from "./Field"
+import TextInput from "./TextInput"
+import RuleField from "./RuleField"
+
+export const BirthField = {
+  props: {
+    value: {
+      type: Object,
+      default: () => ({
+        year: "",
+        month: "",
+        day: "",
+      }),
+    },
+    required: Boolean,
+  },
+  components: {
+    Label,
+    Field,
+    TextInput,
+    RuleField,
+  },
+  data: () => ({
+    year: "",
+    yearRules: [
+      v => !!v || "필수 정보입니다.",
+      v => v.length === 4 || "태어난 년도 4자리를 정확히 입력해 주세요.",
+    ],
+    month: "",
+    monthRules: [
+      v => !!v || "필수 정보입니다.",
+      v => v.length === 2 || "태어난 월 2자리를 정확히 입력해 주세요.",
+    ],
+    day: "",
+    dayRules: [
+      v => !!v || "필수 정보입니다.",
+      v => v.length === 2 || "태어난 일 2자리를 정확히 입력해 주세요.",
+    ],
```

Comment:
```suggestion
    yearRules: [
      v => !!v || "필수 정보입니다.",
      v => v.length === 4 || "태어난 년도를 4자리로 입력해 주세요.",
    ],
    month: "",
    monthRules: [
      v => !!v || "필수 정보입니다.",
      v => v.length === 2 || "태어난 월을 2자리로 입력해 주세요.",
    ],
    day: "",
    dayRules: [
      v => !!v || "필수 정보입니다.",
      v => v.length === 2 || "태어난 일을 2자리로 입력해 주세요.",
    ],
```

7월이나 2일에 태어난 사람의 경우에는 사실 1자리기에 2자리로 입력해달라고 메시지를 띄우는게 더 괜찮겠다는 생각으로 메시지를 고쳐봤습니다. 
핀이 생각하기에 괜찮다면 반영해주세요~